lib/config/definitions.ts

```diff
@@ -510,6 +510,15 @@ const options: RenovateOptions[] = [
     mergeable: true,
     cli: false,
   },
+  {
```

Comment:
If volta is managed by `npm` then there's no need for this. Remove it completely as well as the doc entry for it. Instead, add a description within the npm manager. Alternatively, implement it as a new manager instead (lib/manager/volta).

Reply:
I'll merge it under npm.

Reply:
I just didn't understand this part ->  `Instead, add a description within the npm manager` 
You mean add a description under npm config in definitions? or docs?

Reply:
I mean in the docs, under `## npm` or whatever it is

Reply:
okay, got it!